manifests/m/Mozilla/FirefoxNightly/91.0a1/Mozilla.FirefoxNightly.yaml

```
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: Mozilla.FirefoxNightly
+PackageVersion: 91.0a1

```

```suggestion
PackageVersion: 91.0a1-20210616
```
You'll have to update the version folder too using GitHub Desktop.